src/config/settings.py

```diff
@@ -31,6 +31,17 @@
 
 # Application definition
 
+
+default_renderer_classes = os.getenv(
+    "REST_DEFAULT_RENDERER_CLASSES", "rest_framework.renderers.JSONRenderer"
+)
+REST_FRAMEWORK = {
+    # https://www.django-rest-framework.org/api-guide/renderers/
+    "DEFAULT_RENDERER_CLASSES": list(
+        map(str.strip, default_renderer_classes.split(","))
```

Comment:
Please, instead of `map` use list comprehension `[default_renderer_class.strip() for default_renderer_class in default_renderer_classes.split(",")]` https://blog.finxter.com/which-is-faster-list-comprehension-or-map-function-in-python/

Reply:
A good catch 🙌